frontend/src/DocumentSources.js

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+import FormattedText from './FormattedText';
+
+function DocumentSources({children}) {
+  let sourcesTab = children.split('\n\n');
```

Comment:
Splitting after concatenating is a bit weird (and not very robust since someone could type `\n\n` also).
Instead you should remove both.